docs/snippets/azure/AppHost/Program.ConfigureStorageInfra.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Azure.Provisioning.Storage;
+using Azure.Core;
+
+internal static partial class Program
+{
+    public static void ConfigureStorageInfra(IDistributedApplicationBuilder builder)
+    {
+        // <configure>
+        builder.AddAzureStorage("storage")
+            .ConfigureInfrastructure(infra =>
+            {
+                var resources = infra.GetProvisionableResources();
+
+                var storageAccount = resources.OfType<StorageAccount>()
+                                              .FirstOrDefault(r => r.BicepIdentifier is "storage")
+                    ?? throw new InvalidOperationException("""
```

Comment:
Why is this exception message using raw string literals? Can you make it a normal string literal?

Reply:
It's intentional so that it goes on the line below, we strive to keep the width of the code a certain char-count to avoid horizontal scrolling. I could still achieve that with a non-literal.